### ISSUE TITLE
IA Request: Claim Letter crosslinks and entrypoints

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#81061]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits Management Tools
- **OCTO product owner:** Amy Lai
- **Product name:** Claim Status Tool
- **Product manager:**  Jacob Worrell
- **Slack channel:** [#benefits-management-tools](https://dsva.slack.com/archives/C04KHCT3ZMY)
- **Dedicated content writer on your team (if you have one):** @laurwill 
- **Dedicated a11y specialist on your team (if you have one):** @davidakennedy 
 
## About your initiative

[Original ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79846)

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Currently, Veterans can get to the Claim Letters page and download claim decision letters when their claim is closed. Since we are adding letter types to the Claim Letters page, we don't want to rely on this navigation pattern alone. 

We want to add a persistent link to the Claim Letters page from: 
1. The CST home page `va.gov/track-claims/your-claims`
2. The Letters page `va.gov/records/download-va-letters/letters/letter-list`
3. The VA Disability Compensation page `va.gov/disability`

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We're working on the designs and looking for IA and Content feedback before implementation.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

## Supporting artifacts

We want to add a persistent link to the Claim Letters page from: 

1. The CST home page `va.gov/track-claims/your-claims`
[Mock](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation?type=design&node-id=1250-3487&mode=design&t=yiC68ascXeSamQMX-4)

2. The Letters page `va.gov/records/download-va-letters/letters/letter-list`
[Mock](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation?type=design&node-id=1250-3973&mode=design&t=yiC68ascXeSamQMX-4)

3. The VA Disability Compensation page `https://staging.va.gov/disability/`
[Mock](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation?type=design&node-id=1268-3914&mode=design&t=yiC68ascXeSamQMX-4)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by kristinoletmuskat** [comment]
Hey @skylerschain ! Thanks for sending the mocks along, super helpful. I see that in the VA Benefit Letters page they aren't listed and you're hoping to add them to the download system. Where do the claim letters live today if a user can't download from [VA Benefit Letters](https://www.va.gov/records/download-va-letters/) page?

**Comment by skylerschain** [comment]
👋 

Today Veterans can only get to the Claim Letters page via an alert when a claim is decided. (The reason is we started with only decision letters, so that made sense for a while). 

You can see designs for that [here](https://www.figma.com/file/F8U4wddaFouUPVd4mGBMDI/CST---Evidence-Submission---Spec?type=design&node-id=1124-8264&mode=design&t=2wHXMKb5fWud0de1-4).

**Comment by kristinoletmuskat** [comment]
Ah ok got it! And have ya'll explored adding the claim letters into the VA Benefit Letters tool, instead of having a separate place for this type of letter? I'm guessing there is some sort of technical blocker for doing that?

**Comment by skylerschain** [comment]
Yeah! So if you look in the **Supporting Artifacts** section of this ticket it lays out the different entrypoints where we want to add this link. The second one is the Letters app.

**Comment by kristinoletmuskat** [comment]
Right! I see that, but I'm wondering why we can't add the claim letters into the tool, rather than as a separate download outside of the tool.

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/e2d03caa-3927-4721-8ee4-04c66d3ad926)

@skylerschain

**Comment by skylerschain** [comment]
@kristinoletmuskat Ah I see what you mean. I'd say the reason for that is because this page is about `VA Benefit Letters` which are distinct from `VA Claim Letters`. We did consider combining them originally but decided to house them separately due to some user confusion and for overall simplicity in building out the Claim Letters page. 

Here's a [one-pager](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/claim-appeal-status/research/2022-08-Decision-Letter-Download/letters-strategy-proposal.md) I wrote way back then documenting the that decision and some additional thoughts if you're curious.

**Comment by kristinoletmuskat** [comment]
@skylerschain sweet! The one page is super helpful! I'm gonna talk with Mikki tomorrow because I think the point about designing a global solution for letters is important, I want to see where this is on OCTO's radar. 

But I think in the meantime your solution of adding a bunch of crosslinks everywhere makes sense. Just tagging @laurwill for awareness -- this is the ticket I mentioned in hive mind this am.

**Comment by skylerschain** [comment]
Thanks! And yeah @laurwill let me know if you have feedback and how you'd like to collaborate. I was gonna tag you in Figma but can do whatever works best!

**Comment by skylerschain** [comment]
Also just a heads up: the first part of this we'll be tackling (starting tomorrow in our next sprint) is the link from CST to the Claim Letters page (captured in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team#workspaces/benefits-management-tools-65e8f0cc3b7a040aa567ce58/board)). So if y'all have feedback on that part, let's talk!

**Comment by kristinoletmuskat** [comment]
Hey! yes that's fine -- we sign off on adding the crosslinks from CST homepage and letters page. We don't think we can have a link on the Disability hub page to claim letters UNLESS there is a static page for unauth users, which I don't think there is? 

We also 100% support ya'll doing more research on the letters ecosystem to figure out a better way to consolidate these concepts. 

![Screenshot 2024-04-24 at 9 41 25 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/b5887f57-605c-406d-84ce-bfd7806ad3f9)

**Comment by kristinoletmuskat** [comment]
Hey @skylerschain -- do you want to talk about anything else here or can we close this ticket?

**Comment by skylerschain** [comment]
I think we can close – thanks for your help!
